### DISCUSSION TITLE
Inconsistent ammo type naming and insufficient descriptions (+fixes for it)

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#13704] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Some ammo types don't include the name of the turret in their name, making it impossible to, for example, find all types of coilgun ammunition in the list of fabricator's recipes by simply searching for "Coilgun". Same goes for the railgun ammo (they're all shells, but having "Railgun" in the name would help new players).
Descriptions of railgun shells are also lacking. Most of them just state that the shells are supposed to be used with the railgun, not much else.
Here are the proposed changes:

Exploding Ammunition Box -> Explosive Coilgun Ammunition Box _("Exploding" ammunition is ammo rigged to explode inside the gun. No other ammo type in the game uses that word.)_
Piercing Ammunition Box -> Piercing Coilgun Ammunition Box
Physicorium Ammunition Box -> Physicorium Coilgun Ammunition Box

Railgun Shell
Standard railgun ammunition. Explodes on impact. Can be loaded with additional explosives.

Nuclear Shell -> Nuclear Railgun Shell
A nuclear shell for use with a railgun. Produces a massive nuclear explosion on impact. Can be loaded with additional explosives.

Physicorium Shell -> Physicorium Railgun Shell
A heavy railgun shell made primarily of Physicorium. Deals extremely high direct damage. Can be loaded with additional explosives.

Canister Shell -> Canister Railgun Shell
A reloadable burst-type munition for the railgun. (Description unchanged.)

I also fixed a typo in the description of the tri-laser ammo (extra "of") and a bunch of incorrect uses of it's/its.
This could've been a PR, but there's no real benefit to submitting these changes that way, as far as I can tell.

Edited file:
[EnglishVanilla.zip](https://github.com/FakeFishGames/Barotrauma/files/14814938/EnglishVanilla.zip)

### Reproduction steps

1. Read.
2. :(

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.3.0.0/v1.3.0.1

### Which operating system did you encounter this bug on?

Windows

**Comment by BlackMoons** [comment]
Can confirm, seen a lot of new players who don't know that Exploding Ammunition Box/Piercing Ammunition Box/Physicorium Ammunition Box are for Coilguns. When asked to make/recycle coilgun ammo they often can't find it in the fabricator since its the only ammo that lacks the name of the weapon its for.

**Comment by Regalis11** [comment]
This has now been addressed in our private repo, the changes are coming up in the next update.
